src/train.py

```diff
@@ -100,113 +100,6 @@ def train_model(settings, hyp_params, train_loader, valid_loader, test_loader):
         else:
             model = model.cuda()
 
-    def train(model, optimizer, criterion, local_rank, reg_en=False):
```

Comment:
move inner-function out of the main function. To reduce the size of main function.